### PR TITLE
Java: Updating LongRunning

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

File: generated/java/google-longrunning-v1/src/main/java/com/google/longrunning/OperationsClient.java [hunk 1]
```diff
@@ -198,16 +201,17 @@ public final OperationsSettings getSettings() {
...
+   *   String formattedName = OperationsClient.formatOperationPathName("[OPERATION_PATH]");
+   *   Operation response = operationsClient.getOperation(formattedName);
    * }
    * </code></pre>
    *
    * @param name The name of the operation resource.
    * @throws com.google.api.gax.grpc.ApiException if the remote call fails
    */
   public final Operation getOperation(String name) {
+    OPERATION_PATH_PATH_TEMPLATE.validate(name, "getOperation");
```
Comment (michaelbausor) [inline generated/java/google-longrunning-v1/src/main/java/com/google/longrunning/OperationsClient.java]: Is it the case that the operation name must always be formatted this way? The comment in the proto seems to hedge a bit: https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto#L84

Comment (garrettjonesgoogle) [inline generated/java/google-longrunning-v1/src/main/java/com/google/longrunning/OperationsClient.java]: Good catch - I think we should remove the collection configuration for this - https://github.com/googleapis/googleapis/blob/master/google/longrunning/longrunning_gapic.yaml#L24 - I'll send a PR for it.

Comment (garrettjonesgoogle) [inline generated/java/google-longrunning-v1/src/main/java/com/google/longrunning/OperationsClient.java]: https://github.com/googleapis/googleapis/pull/201

**Comment by garrettjonesgoogle** [comment]
Updated, PTAL

**Comment by michaelbausor** [comment]
LGTM

**Pull request merged**